src/validation.js

```diff
@@ -0,0 +1,30 @@
+const validation = (text, toCase) => {
```

Comment:
The function name 'validation' is not a verb or verb phrase. Consider renaming it to something that better describes its action like 'validateTextAndCase'.